Sources/StreamChat/APIClient/Endpoints/MessageEndpoints.swift

```diff
@@ -126,3 +136,39 @@ extension Endpoint {
         )
     }
 }
+
+public struct MessageReactionsQueryFilter: Encodable {
+    enum CodingKeys: String, CodingKey {
+        case type
+        case authorId = "user_id"
+    }
+
+    public var type: MessageReactionType?
+    public var authorId: UserId?
+
+    public init(type: MessageReactionType? = nil, authorId: UserId? = nil) {
+        self.type = type
+        self.authorId = authorId
+    }
+}
```

Comment:
This needs to be changed to our `Filter` type to make it consistent with the rest of the queries.